nagios/README.md

```diff
@@ -51,39 +101,99 @@ define contact{
+    command_name nagios-to-wavefront-host
+    command_line /usr/local/nagios/libexec/nagios-events-wf.py --type '$NOTIFICATIONTYPE$' \
+                                                        --host '$HOSTNAME$' --time '$TIMET$' \
+                                                        --msg '$SERVICEOUTPUT$\n$NOTIFICATIONAUTHOR$\n$NOTIFICATIONCOMMENT$' \
+                                                        --wf_server [WAVEFRONT_URL] \
+                                                        --csp_app_id [CSP_APP_ID] \
+                                                        --csp_app_secret [CSP_APP_SECRET] \
+                                                        --csp_base_url [CSP_BASE_URL]
 }
+
+
+define command {
+    command_name nagios-to-wavefront-service
+    command_line /usr/local/nagios/libexec/nagios-events-wf.py -S --type '$NOTIFICATIONTYPE$' \
+                                                        --service '$SERVICEDISPLAYNAME$' \
+                                                        --host '$HOSTNAME$' --time '$TIMET$' \
+                                                        --msg '$SERVICEOUTPUT$\n$NOTIFICATIONAUTHOR$\n$NOTIFICATIONCOMMENT$' \
+                                                        --wf_server [WAVEFRONT_URL] \
+                                                        --csp_app_id [CSP_APP_ID] \
+                                                        --csp_app_secret [CSP_APP_SECRET] \
+                                                        --csp_base_url [CSP_BASE_URL]
+}
+ ```
+
+#### Using Wavefront Ingestion
 ```
+define command {
+    command_name nagios-to-wavefront-host
+    command_line /usr/local/nagios/libexec/nagios-events-wf.py --type '$NOTIFICATIONTYPE$' \
+                                                       --host '$HOSTNAME$' \
+                                                       --time '$TIMET$' \
+                                                       --msg '$HOSTOUTPUT$' \
+                                                       --wf_server [WAVEFRONT_URL] \
+                                                       --wf_token [WF_API_TOKEN]
+}
+
+
+define command {
+    command_name nagios-to-wavefront-service
+    command_line /usr/local/nagios/libexec/nagios-events-wf.py -S --type '$NOTIFICATIONTYPE$' \
+                                                       --service '$SERVICEDISPLAYNAME$' \
+                                                       --host '$HOSTNAME$' --time '$TIMET$' \
+                                                       --msg '$SERVICEOUTPUT$\n$NOTIFICATIONAUTHOR$\n$NOTIFICATIONCOMMENT$' \
+                                                       --wf_server [WAVEFRONT_URL] \
+                                                       --wf_token [WF_API_TOKEN]
+}
+```
+
+Refer the configuration samples at https://github.com/wavefrontHQ/integrations/raw/master/nagios/config to send events using CSP API token, CSP Oauth or Wavefront API Token.
```

Comment:
...using a VMware Cloud services API token, server to server OAuth app credentials, or an Operations for Applications API token.